i18n/fr.yaml

```diff
@@ -5,74 +5,62 @@ twitter_description:
 title:
   other: "COVID Shield"
 intro:
-  other: "COVID Shield is a free exposure notification solution built
-    with privacy as its top priority. It was built by a group of
-    volunteers in order to help Canadians and the rest of the world safely
-    return to work."
+  other: "COVID Shield est une solution de notification d'exposition libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité."
 disclaimer_title:
   other: "How can I download and use COVID Shield?"
 disclaimer_message:
   other: "COVID Shield is not currently available for download or use."
 disclaimer_content:
   other: "COVID Shield is provided as a reference for your local public health authority to build their own app. If you are interested in using COVID Shield, please contact your local government representative and ask what they are doing about exposure notification for COVID-19."
 view_on_github:
-  other: "View on GitHub"
+  other: "Voir sur GitHub"
 contact_us:
-  other: "Contact us"
+  other: "Nous contacter"
 paragraph_1:
-  other: 'COVID Shield is a private, secure, and easy-to-use tool to help
-    governments launch their own exposure notification systems. It is
-    based on the exposure notification technology <a
-    href="https://www.apple.com/covid19/contacttracing">provided by Apple
-    and Google</a>, the most privacy-preserving approach currently
-    available.'
+  other: 'COVID Shield est un outil privé, sécurisé et facile à utiliser, qui vise à aider les gouvernements à lancer leurs propres systèmes de notification d'exposition. L'outil est basé sur une technologie de notification d'exposition <a
+    href="https://www.apple.com/covid19/contacttracing">fournie par Apple et Google</a>, dont l'approche scrupuleuse en matière de confidentialité n'a pas d'équivalent sur le marché actuel.'
 paragraph_2:
-  other: "For exposure notification to be most effective, it is important that
-    governments launch applications that can work together. COVID Shield
-    is flexible enough to be deployed at various levels of government by
-    enabling the server and mobile app to talk to each other even if there
-    are multiple instances."
+  other: "Pour que l'efficacité du système de notification d'exposition soit optimale, il est important que les gouvernements lancent des applications compatibles les unes avec les autres. COVID Shield est suffisamment flexible pour être déployé à différents ordres de gouvernement en activant le serveur et l'application mobile de manière qu'ils communiquent même en présence de plusieurs instances."
```

Comment:
We want to use "notification d'exposition **à la COVID-19**" as we did in the app so I suggest:
`Pour que l'efficacité du système de notification d'exposition à la COVID-19 soit optimale, il est important que les gouvernements lancent des applications compatibles les unes avec les autres. COVID Shield est suffisamment flexible pour être déployé à différents ordres de gouvernement en activant le serveur et l'application mobile de manière qu'ils communiquent même en présence de plusieurs instances.`